sync-with-master/entrypoint.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -eux
+
+#
+# This variable must be passed into this script.
+#
+BRANCH="$1"
+[[ -n "${BRANCH}" ]] || exit 1
+
+#
+# We need these config parameters set in order to do the git-merge.
+#
+git config user.name "${GITHUB_ACTOR}"
+git config user.email "${GITHUB_ACTOR}@users.noreply.github.com"
+
+#
+# We need the full git repository history in order to do the git-merge.
+#
+git fetch --unshallow
+
+#
+# In order to open a pull request, we need to push to a remote branch.
+# To avoid conflicting with existing remote branches, we use branches
+# within the "sync-with-master" namespace.
```

Comment:
Looks like a mismatch between the name of the branch in the comment vs the code. I'm fine either way, but let's make sure they match.

Reply:
Good catch; let me fix that.